Add User Metrics skeleton #116

alouanemed opened the pull request:
here is the final result

![image](https://cloud.githubusercontent.com/assets/7117097/5898984/4df07fc8-a54d-11e4-81e0-14a51fdf8234.png)


You can access this activity using the sign up button .

intrications commented on app/src/main/java/com/kanshu/kanshu/UserMetricsActivity.java:
```diff
@@ -0,0 +1,147 @@
+
+    public class UserMetrcisPagerAdapter extends FragmentStatePagerAdapter {
+
+        private String[] pageTitles = {"My Prgoress", "My saved Characters", "Practice Exercices", "Add-ons"};
```
Correct spelling to Progress
Saved should start with a capital 's'
All strings should be extracted to strings.xml ready for translation.

intrications commented on app/src/main/java/com/kanshu/kanshu/UserMetricsActivity.java:
```diff
@@ -0,0 +1,147 @@
+    }
+
+
+    public class UserMetrcisPagerAdapter extends FragmentStatePagerAdapter {
```
Metrcis > Metrics

intrications commented on app/src/main/java/com/kanshu/kanshu/UserMyPrgoressFragment.java:
```diff
@@ -0,0 +1,49 @@
+/**
+ * A placeholder fragment containing a simple view.
+ */
+public class UserMyPrgoressFragment extends Fragment {
```
Spelling of Progress

intrications commented on app/src/main/java/com/kanshu/kanshu/UserPracticeExofragment.java:
```diff
@@ -0,0 +1,40 @@
+     */
+    private static final String ARG_SECTION_NUMBER = "section_number";
+
+    public static UserPracticeExofragment newInstance(int sectionNumber) {
```
Fragment should be capitalised

intrications commented on app/src/main/res/layout/fragment_user_my_saved_chars.xml:
```diff
@@ -0,0 +1,17 @@
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textAppearance="?android:attr/textAppearanceLarge"
+        android:text="My Saved Chars"
```
Extract to strings.xml - should be already there from the matching tab name.

intrications commented:
Looks good.

Can you fix these issues?

Also, have you replaced the contents of ArticlesActivity? If so, can you create a new activity for this?

Then I will add a screen which links to all the Activities that are being created rather than each developer taking over the sign up button.

Thanks.

intrications commented:
Also, it probably needs rebasing on master.

alouanemed commented:
@intrications all fixed;
I updated ArticlesActivity's layout name which is used in UserMetricsActivity , they both have the same content.

that's weird I rebased before pushing this commits !

intrications commented:
It's still got the old style navigation drawer that why I noticed that I don't think it was rebased.

intrications commented on app/src/main/java/com/kanshu/kanshu/ArticleActivity.java:
```diff
@@ -34,7 +32,7 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_article);
+        setContentView(R.layout.activity_main);
```
Why was this layout name changed?

intrications replied:
Oh, I see because they both have the same content.

intrications replied:
main doesn't seem right for something that is reused. Can't think of a better name though. activity_with_nav_drawer_and_tabs seems too long.

alouanemed replied:
main layout for now until we found something good !

intrications replied:
Agreed.

alouanemed commented:
cc @intrications  ;all good ?

fernandobuzi commented:
@alouanemed @intrications Is it possible to have all the tab titles visible? Is it like that because the titles are too long?

Should we, maybe, try Progress/Saved Characters/Practice/Add-ons?

fernandobuzi commented:
Unless the selected tab always aligns to the left? If that is so, I think it can stay as it is now.

intrications commented:
@fernandobuzi There's no point trying to fit all the tabs in. Android phones go down to 3 inches and smaller. I'm just about to test the code but I suspect the tabs may left aligned like that.

fernandobuzi commented:
;)

intrications commented:
@alouanemed Have you checked your build works? Have you pushed all the files? I'm getting complaints about missing strings in strings.xml in UserMetricsActivity.

intrications commented:
Also the actual UserPracticeExoFragment file does not appear to have been renamed. Only the declaration of class.
